### DISCUSSION TITLE
Can't enroll new resource in teleport & self sign certificate error in teleport

### DISCUSSION THREAD

**Discussion by Pr0t0c01** [gravitational/teleport#35288] [Q&A]
Hello everyone, 
  Now, I'm trying to connect teleport server (kali) and new resource (ubuntu server 14.04). But i'm facing with certificate has expired. I have installed teleport server in my kali. But when i enroll new resouce (ubuntu server 14.04), 
- sudo bash -c "$(curl -k -fsSL https://shiroeteleport.com/scripts/blahblah/install-node.sh)"

I got this error, curl: (60) ssl certificate problem: certificate has expired.

My self sign certificate config
====================
┌──(shiroe㉿kali)-[~/teleport]
└─$ cat san.cnf                                    
[req]
distinguished_name = req_distinguished_name
req_extensions = v3_req
prompt = no

[req_distinguished_name]
CN = shiroeteleport.com

[v3_req]
keyUsage = keyEncipherment, dataEncipherment
extendedKeyUsage = serverAuth
subjectAltName = @alt_names

[alt_names]
DNS.1 = shiroeteleport.com
DNS.2 = www.shiroeteleport.com

/etc/teleport.yml
============
┌──(root㉿kali)-[/var/lib/teleport]
└─# cat /etc/teleport.yaml     
version: v3
teleport:
  nodename: kali
  data_dir: /var/lib/teleport
  log:
    output: stderr
    severity: INFO
    format:
      output: text
  ca_pin: ""
  diag_addr: ""
auth_service:
  enabled: "yes"
  listen_addr: 0.0.0.0:3025
  cluster_name: shiroeteleport.com
  proxy_listener_mode: multiplex
ssh_service:
  enabled: "yes"
  commands:
  - name: hostname
    command: [hostname]
    period: 1m0s
proxy_service:
  enabled: "yes"
  web_listen_addr: 0.0.0.0:443
  public_addr: shiroeteleport.com:443
  https_keypairs:
  - key_file: /var/lib/teleport/teleport.key
    cert_file: /var/lib/teleport/teleport.pem
  https_keypairs_reload_interval: 0s
  acme: {}

**Comment by webvictim** [comment]
What does `openssl x509 -in /var/lib/teleport/teleport.pem -text | grep "Not "` say?
